.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,96 @@
+name: "🐛 Bug Report"
+description: "If something isn't working as expected 🤔."
+title: "[Bug]: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: "## Please note the following potential times when an issue might be in Terraform core:"
+  - type: markdown
+    attributes:
+      value: |
+        * [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
+        * [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
+        * [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
+        * [Registry](https://registry.terraform.io/) issues
+        * Spans resources across multiple providers
+
+        If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
+  - type: textarea
+    attributes:
+      label: Community Note
+      description: Please leave this note as is for the community
+      value: |
+        * Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+        * Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+        * If you are interested in working on this issue or have submitted a pull request, please leave a comment
+    validations:
+      required: true
```

Comment:
This is the only part which I don't love, but it doesn't seem like the [GitHub Form Schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema) allows you to add content to the issue without a editable form element. Just an idea, but I wonder if instead of keeping the note as part of the the issue body, we could add a GitHub action to add the content as a comment on creation?

Reply:
In case we decide to go that route, looks like [this action](https://github.com/marketplace/actions/create-or-update-comment) would make it really easy to make that happen.